### PR TITLE
CSIP8: Create testCase.xml

### PR DESCRIPTION
**Pull request by koit**
#85

### CODE REVIEW

**Review by PhillipTommerholt — CHANGES_REQUESTED** [review]
I don't think we could know if the package has been modified or not, so I would suggest that rule 1 is untestable. 
If there exist different representations and they have different creation dates, then this test could make sense, but I think that a test like that is too advanced to spend time on right now - it could be a future test.

**Comment by koit** [comment]
> I don't think we could know if the package has been modified or not, so I would suggest that rule 1 is untestable.
> If there exist different representations and they have different creation dates, then this test could make sense, but I think that a test like that is too advanced to spend time on right now - it could be a future test.

I agree, it's clearly untestable and I see two solutions: a) remove the rule, or b) issue an info or warning message. I don't have a clear preference - each solution has its problems. As option b is currently implemented, the easiest is to leave it as it is and update if strong arguments in favour of the alternative appear 🙂.

File: corpora/csip/metadata/metshdr/CSIP8/mets-xml_metsHdr_LASTMODDATE_in_future/METS.xml [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    http://www.w3.org/1999/xlink http://www.loc.gov/standards/xlink/xlink.xsd"
+  OBJID="minimal_IP_with_1_representation" 
+  TYPE="Mixed"  
+  PROFILE="https://earkcsip.dilcis.eu/profile/E-ARK-CSIP.xml">
+
+  <!-- CSIP117 mets/metsHdr - Package header - [1..1] -->
+  <!-- CSIP7   mets/metsHdr/@CREATEDATE - Package creation date - [1..1] -->
+  <!-- CSIP8   mets/metsHdr/@LASTMODDATE - Package last modification date - [0..1] -->
+  <!-- CSIP9   mets/metsHdr/@csip:OAISPACKAGETYPE - OAIS Package type information - [1..1] -->
+  <metsHdr CREATEDATE="2019-04-14T20:00:00" csip:OAISPACKAGETYPE="SIP">
```
Comment (PhillipTommerholt) [inline corpora/csip/metadata/metshdr/CSIP8/mets-xml_metsHdr_LASTMODDATE_in_future/METS.xml:27]: I don't see a LASTMODDATE?

**Review by PhillipTommerholt — CHANGES_REQUESTED** [review]
the package with a moddate in the future does not seem to have an moddate-attrbiute?

File: corpora/csip/metadata/metshdr/CSIP8/mets-xml_metsHdr_LASTMODDATE_in_future/METS.xml (hunk 1 above)
Comment (koit) [reply corpora/csip/metadata/metshdr/CSIP8/mets-xml_metsHdr_LASTMODDATE_in_future/METS.xml:27]: Sorry, I forgot to say that the focus currently was only on testCase.xml, I didn't even notice CSIP8 had packages done. Ülle created new packages based on the valid package `valid_IP_with_SHOULD_MAY_1_rep` and these handle LASTMODDATE correctly.

In sum, let's only fix testCase.xml now, then I merge the PR, then Ülle uploads the correct packages and creates a new PR.

**Review by PhillipTommerholt — APPROVED** [review]

**Pull request merged**